digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "NotesMaster",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://notesmaster.com/termsofuse"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "The content is authored using the Tiny MCE Editor which is a WYSISWYG editor based on HTML. This makes it versatile in the insertion of links and the embedding of other learning software such as H5P, Prezi, Slideshare, nearpod, Wordwall, Vyond, Kazoo, Quizizz, Storyboard That, Pixton. The platform operates across all browsers, is mobile responsive and allows all content to be printed and downloaded to MS Word Format. The Notesmaster YouTube channel guides users in how to register, find classes and generally get started:https://www.youtube.com/channel/UCBFmZoPITZFZl-DFYbx-2nA/videos. This support link is accessible on the website by clicking the ‘?’ at the top."
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Federal Children’s Online Privacy Protection Act of 1998",
+      "U.S. Family Educational Rights and Privacy Act "
+    ],
+    "adherenceSteps": [
+      "https://notesmaster.com/privacy"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "W3C Accessibility Standards",
+      "WCAG 2.1 standards",
+      "Creative Commons standards"
+    ],
+    "evidenceStandardSupport": [
+      "https://www.tiny.cloud/docs/release-notes/release-notes50/#uiaccessibility",
+      "https://www.tiny.cloud/docs/configure/content-filtering/#convert_fonts_to_spans",
+      "Please see reference to our Terms of Use which highlights our adherence to the Creative Commons Standards, as well as our adherence to the the Federal Children’s Online Privacy Protection Act of 1998 (“COPPA”) in the United States and the European Union’s General Data Protection Regulation (GDPR) of 2018: https://notesmaster.com/termsofuse"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Open access to content",
```

Comment:
Is the content openly accessible outside the NotesMaster platform? The content seems to be `free`, but always behind a login, which does not make it `open`. Please clarify.

Reply:
Here are some links to resource on Notesmaster which you can access without login:

- ​Primary English in Trinidad and Tobago  https://old.notesmaster.com/en/group/caribbean/1464-moe-oer-tt-group/47869-listening-behaviours
- Grade 10 Physics from Mozambique (in Portuguese) https://old.notesmaster.com/en/group/mozambique/2-col-ined-oer/814-identificar-o-tipo-de-interacao-entre-os-corpos-electricamente-carregados
- Grade 9 Biology from Mozambique in (Portuguese): https://old.notesmaster.com/en/group/mozambique/2-col-ined-oer/1279-consolidar-os-conhecimentos-sobre-a-celula-vegetal​
- Senior Secondary Mathematics from Belize: https://old.notesmaster.com/en/group/caribbean/1552-belize-aptus-pilot-group/30969-aptus-removing-double-signs
- Geography from Malawi: https://old.notesmaster.com/en/group/malawi/25-moe-malawi-covid19-support-group/310-components-of-the-solar-system
- Chichewa from Malawi: https://old.notesmaster.com/en/group/malawi/25-moe-malawi-covid19-support-group/584-ntchito-za-aphatikiram-tsogolo

Reply:
These links are great. My follow up question would be how can someone access this content without knowing the direct links. Are these the result of a search, or is there a list or a table of contents somewhere?

Reply:
This issue has been adequately resolved, thank you.